services/src/main/scala/cromwell/pipeline/service/GitLabProjectVersioning.scala

```diff
@@ -225,14 +225,15 @@ class GitLabProjectVersioning(httpClient: HttpClient, config: GitLabConfig)
     }
 
     httpClient
-      .get[ProjectFileContent](
-        s"${config.url}/projects/${project.repositoryId.value}/repository/files/$filePath/raw",
+      .get[GitLabFileContent](
+        s"${config.url}/projects/${project.repositoryId.value}/repository/files/$filePath",
         Map("ref" -> fileVersion),
         config.token
       )
       .map {
-        case Response(HttpStatusCodes.OK, SuccessResponseBody(projFileContent), _) =>
-          Right(ProjectFile(path, projFileContent))
+        case Response(HttpStatusCodes.OK, SuccessResponseBody(gitLabFile), _) =>
+          val content = new String(java.util.Base64.getDecoder.decode(gitLabFile.content))
```

Comment:
I think you should create a method like
`private def decodeBase64(str: String): String = ???`
It will make code a little bit easier to read and will allow us to override this method in tests if necessary

Reply:
Method created.